### PR TITLE
Listing seperate extension for VS Code.

### PR DESCRIPTION
**Pull request by mitchdenny**
VS Code also has an extension for ```.editorconfig```. I've added a second item in the list of plug-ins to install and changed the colour of the logo (VS Code uses a blueish logo instead of the purpose one). I've also credited the author of the VS Code plug-in.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 _site
+.DS_Store
```
Comment (johan) [inline .gitignore]: If you're adding a new line, it's good form to add the newline character after it too, so git tools won't complain like this.

Comment (johan) [inline .gitignore]: (Interesting – github apparently only shows that warning in whole-file-view mode, not when looking at the inline pull request comment here.)

**Comment by jedmao** [comment]
:+1:

File: .gitignore (hunk 1 above)
Comment (xuhdev) [inline .gitignore]: And this line change should prbably be in a separate commit :) can you update the commit? You can do so by run `git rebase --interactive master`.

**Comment by mitchdenny** [comment]
I've backed out the ```.gitignore``` change. Did you want me to squash the two commits as well?

**Comment by xuhdev** [comment]
Thanks, please do so.

**Comment by mitchdenny** [comment]
Done. We ready to pull?

**Comment by xuhdev** [comment]
thanks!
